Resources/config/routing_pjax.yml

```diff
@@ -147,7 +147,7 @@ admin_roleList:
         _controller: ezsystems.platformui.controller.role:listRolesAction
 
 admin_roleView:
-    path: /role/view/{roleId}
+    path: /role/view/{roleId}/{tabId}
```

Comment:
http://symfony.com/doc/current/book/routing.html#required-and-optional-placeholders